styles/morning.json

```diff
@@ -0,0 +1,338 @@
+					"slug": "primary"
+				},
+				{
+					"color": "#D7D3CC",
+					"name": "Secondary",
+					"slug": "secondary"
+				},
+				{
+					"color": "#19191933",
+					"name": "Opacity 20%",
+					"slug": "opacity-20"
+				}
+			]
+		}
+	},
+	"styles": {
+		"color": {
+			"text": "var:preset|color|primary"
+		},
+		"typography": {
+			"fontFamily": "var:preset|font-family|ysabeau-office",
+			"fontSize": "1.5rem",
+			"fontStyle": "normal",
+			"fontWeight": "400",
+			"letterSpacing": "-0.24px",
+			"lineHeight": "1.5"
+		},
+		"blocks": {
+			"core/button": {
+				"variations": {
+					"outline": {
+						"border": {
+							"color": "currentColor"
+						}
+					}
+				}
+			},
+			"core/code": {
+				"color": {
+					"text": "var:preset|color|contrast",
+					"background": "var:preset|color|secondary"
+				}
+			},
+			"core/navigation": {
+				"typography": {
+					"fontSize": "1.25rem"
+				}
+			},
+			"core/paragraph": {
+				"elements": {
+					"link": {
+						"color": {
+							"text": "var:preset|color|contrast"
+						}
+					}
+				}
+			},
+			"core/post-author-name": {
+				"typography": {
+					"fontSize": "var:preset|font-size|medium"
+				},
+				"color": {
+					"text": "var:preset|color|contrast"
+				},
+				"elements": {
+					"link": {
+						"color": {
+							"text": "var:preset|color|contrast"
+						},
+						"typography": {
+							"textDecoration": "underline"
+						},
+						":hover": {
+							"typography": {
+								"textDecoration": "none"
+							}
+						}
+					}
+				}
+			},
+			"core/post-terms": {
+				"typography": {
+					"fontWeight": "400",
+					"fontSize": "var:preset|font-size|medium"
+				},
+				"color": {
+					"text": "var:preset|color|contrast"
+				},
+				"elements": {
+					"link": {
+						"color": {
+							"text": "var:preset|color|contrast"
+						},
+						"typography": {
+							"textDecoration": "underline"
+						},
+						":hover": {
+							"typography": {
+								"textDecoration": "none"
+							}
+						}
+					}
+				}
+			},
+			"core/post-title": {
+				"typography": {
+					"fontWeight": "900",
+					"letterSpacing": "-0.96px"
+				},
+				"elements": {
+					"link": {
+						"color": {
+							"text": "var:preset|color|contrast"
+						}
+					}
+				}
+			},
+			"core/pullquote": {
+				"color": {
+					"text": "var:preset|color|contrast"
+				},
+				"typography": {
+					"fontSize": "var:preset|font-size|x-large",
+					"letterSpacing": "-0.76px"
+				},
+				"elements": {
+					"cite": {
+						"typography": {
+							"fontSize": "1.12rem",
+							"letterSpacing": "-0.14px"
+						},
+						"color": {
+							"text": "var:preset|color|primary"
+						}
+					}
+				}
+			},
+			"core/quote": {
+				"color": {
+					"text": "var:preset|color|contrast"
+				},
+				"typography": {
+					"fontSize": "1.5rem",
+					"fontWeight": "500",
+					"letterSpacing": "-0.18px"
+				},
+				"elements": {
+					"cite": {
+						"typography": {
+							"fontSize": "1.12rem",
```

Comment:
I believe this needs to use em if it is supposed to be relative to the size of the quote.

Reply:
The intention was to have it in `rem` to match the design (~18px). Now that I see this, we can use the medium preset. Unless you prefer to have the `em` value.

<img width="1757" alt="Screenshot 2024-09-24 at 11 24 56" src="https://github.com/user-attachments/assets/4c9d8559-f015-4da2-abe8-1ce462f0acce">